create_text_to_image_ds.py

```diff
@@ -0,0 +1,117 @@
+from ann_benchmarks.algorithms.bruteforce import BruteForceBLAS
+import struct
+import numpy as np
+import click
+import h5py
+from joblib import Parallel, delayed
+import multiprocessing
+
+def read_fbin(filename, start_idx=0, chunk_size=None):
```

Comment:
consider having `float` and `int` as parameters?

Reply:
why?

Reply:
Im not sure why we need the two functions and not only float, but it is code duplication beside the type, and maybe in the future we will want to support more types